docs/src/developer/install.md

```diff
@@ -1,36 +1,118 @@
-# Installing miniSASS Django-CMS project
+# Setting up a Local Development Environment
 
-This is currently a basic Django-CMS site. No extra Django applications have been added yet. 
-To install the pre-requisites needed for this site to run, do the following:
+## Prerequisites
 
-## Docker Deployments
+- Any Linux distribution for the operating system
+- Docker installed on the local machine
+- Visual Studio Code for the editor (or any preferred)
 
-* Clone the repository
+## Installation Steps
 
+1. **Open the terminal.**
+
+2. **Create a directory:**
+    ```bash
+    mkdir name_of_directory
+    ```
+
+3. **Clone the minisass repository:**
+    ```bash
+    git clone https://github.com/kartoza/miniSASS.git
+    ```
+
+4. **Navigate into the repo directory:**
+    ```bash
+    cd miniSASS
+    ```
+
+5. **Open the directory with VSCode (assuming you have VSCode installed) or open the repository directory in any editor of your choice.**
+
+6. **Optional: Adjust the build context of the S3 mount.**
+    In case of build failure, replace lines 32-36 in the `docker-compose.yml` file with the following:
+    ```yaml
+    mount:
+      image: kartoza/s3mount
+      build:
+        context: .
+        dockerfile: ./deployment/docker-s3-bucket/Dockerfile
+    ```
+
+7. **Build the container:**
     ```bash
-     git clone git@github.com:kartoza/miniSASS.git
-     cd miniSASS
+    docker compose build
     ```
 
-* Copy the `.example.env` file and adjust the environment variables to match your requirements.
+8. **Start the containers:**
+    ```bash
+    docker compose up -d db; sleep 180; docker compose up -d
+    ```
 
+9. **Access the Django container:**
     ```bash
-    cp .example.env .env
+    docker exec -it name_of_django_container bash
     ```
+    - Navigate to minisass frontend folder `cd minisass_frontend`
+    - Run `npm install`
+    - Run `npm run build`
+    - Exit the container with 'exit'
+    - Execute the collect static command on the container:
+        ```bash
+        docker exec -it <container_id_or_name> python manage.py collectstatic --noinput
+        ```
+    - Restart the container with `docker restart name_of_django_container`
 
-* Prepare the sample data to load into your PostgreSQL database. If you need to load
-the data when initialising the containers you need to update the `docker-compose.yml`
-so that you can mount the `import-data.sh` script instead of the `restore.sh`
-script as per the instruction of running scripts in [docker-postgis](https://github.com/kartoza/docker-postgis#running-sql-scripts-on-container-startup)
-* Bring the services up by running:
+10. **Access the web application:**
+    Visit 'http://localhost:61122/'
 
+## Local Development (Backend)
+
+- The local database will be empty; fixture files need to be manually loaded for some test data.
+
+### Useful Commands on the Backend Container
+
+- **Creating a superuser:**
```

Comment:
Can we not setup a default user in the image using `python manage.py loaddata ` that way we can control this with env variables